linux/packaging/rpm/make_config.yaml

```diff
@@ -1,38 +0,0 @@
-icon: assets/spotube-logo.png
-summary: Lightweight cross-platform Spotify desktop client
-group: Applications/Multimedia
-vendor: Kingkor Roy Tirtho
-packager: Kingkor Roy Tirtho
-packagerEmail: krtirtho@gmail.com
-license: BSD-4-Clause
-url: https://github.com/KRTirtho/spotube
-requires:
-  - mpv
-  - libappindicator
-  - jsoncpp
-  - libsecret
-  - libnotify
-  - xdg-user-dirs
-  - avahi
-  - mdns-scan
-  - nss-mdns
-  - webkit2gtk4.1
-  - libsoup3
-
-display_name: Spotube
-
-keywords:
-  - Music
-  - Spotify
-  - Media
-  - Streaming
-  - YouTube
-
-generic_name: Music Streaming Application
-categories:
-  - Music
-
-startup_notify: true
-
-supported_mime_type:
-  - x-scheme-handler/spotify
```

Comment:
Again, why the necessity to remove this file? It's important to generate a `.rpm, ` for linux platform